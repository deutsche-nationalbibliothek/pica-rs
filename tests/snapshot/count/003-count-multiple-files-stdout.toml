bin.name = "pica"
args = "count math.dat.gz algebra.dat"
status = "success"
stderr = ""
