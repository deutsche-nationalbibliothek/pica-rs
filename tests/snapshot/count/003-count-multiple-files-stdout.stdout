records: 2
fields: 55
subfields: 114
